src/BigBlueButton.php

```diff
@@ -566,7 +566,7 @@ private function sendRequest(string $url, string $payload = '', string $contentT
             curl_setopt($ch, CURLOPT_POSTFIELDS, $payload);
             curl_setopt($ch, CURLOPT_HTTPHEADER, [
                 'Content-type: ' . $contentType,
-                'Content-length: ' . strlen($payload),
+                'Content-length: ' . mb_strlen($payload),
```

Comment:
It looks like we will have to tackle this down once for all.
Can we get some inspiration from this repository https://github.com/kubawerlos/php-cs-fixer-custom-fixers and create a special custom fixer for this line?

It should be doable with something like this in the custom fixer class.

```php
public function fix(\SplFileInfo $file, Tokens $tokens)
{
    // Replace the line 'Content-length: ' . strlen($payload), with 'Content-length: ' . mb_strlen($payload),
    foreach ($tokens as $index => $token) {
        if ($token->isGivenKind(T_CONSTANT_ENCAPSED_STRING) && $token->getContent() === "'Content-length: ' . strlen(\$payload),") {
            $tokens[$index] = new Token([T_CONSTANT_ENCAPSED_STRING, "'Content-length: ' . mb_strlen(\$payload),"]);
        }
    }
}
```

Reply:
done